Editors/vscode/.vscodeignore

```diff
@@ -1,9 +1,6 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-out/**/*.map
-src/**
+**/*.ts
```

Comment:
Why this instead of `src/`?

Reply:
Recommendation from the VSCode documentation.  I don't have a strong opinion on this.

Reply:
Huh, I was looking at https://code.visualstudio.com/api/working-with-extensions/bundling-extension which suggests "src/", but it's not a big deal either way I guess.

Reply:
https://code.visualstudio.com/api/working-with-extensions/publishing-extension#.vscodeignore is what I was looking through.  You're right that they accomplish more or less the same thing in our particular case.  The difference is that any non-type-script resources would be bundled with the new filter.  I think that if we decide that this is not the right thing for this project, we can easily change this.